test/PowerTools.Test/packages.config

```diff
@@ -1,5 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="Moq" version="4.0.10827" />
+  <package id="Moq" version="4.0.10827" requireReinstallation="True" />
```

Comment:
This was put in automatically when I updated to NET 4.5. But I'm not tied to it so if you think we need to get rid of it so be it.